Update Sparse_matrix_Operation.py

Yash202000 opened the pull request:
In this code, I added the addition and subtraction program code.
I hope this might helpful.

<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

Yash202000 committed:
Update Sparse_matrix_Operation.py

In this code I added addition and subtraction program code.

Yash202000 committed:
Update Sparse_matrix_Operation.py

In this code, I added the addition and subtraction program code. with input format.

Yash202000 commented:
Here in this code I added the addition and subtraction program with simple algorithm.

Yash202000 committed:
Update Sparse_matrix_Operation.py

well indented.

Yash202000 committed:
Update Sparse_matrix_Operation.py

some more indentation errors resolved.

Yash202000 committed:
Update Sparse_matrix_Operation.py

1 bug fix

Yash202000 committed:
Update Sparse_matrix_Operation.py

long line correction

Yash202000 committed:
Update Sparse_matrix_Operation.py

long line check

Yash202000 committed:
Update Sparse_matrix_Operation.py

some changes made

Yash202000 committed:
Update Sparse_matrix_Operation.py

corrected unused variable

Yash202000 commented:
here in this code I added addition program which will store sorted output in list and display it.
hope this might helpful!!

Yash202000 commented:
I am part of DWOC

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
-
-Time Complexity:O(noOfRows*noOfcolumns)
-Space Complexity:O(1)
+1 0 0
```
Add description of the program at the top in commented form.
Add sample output, space and time complexity too and shift this doc string to the end of the program.

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
-if(count == size):
-    print("Null Mtrix")
+# this read_mat function will read input from user and return matrix as list of inputs
 
```
Do not leave blank lines between comment and code.

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
+
+
+def substractspm(sp1, sp2):
+    print("performing substraction on given sparse matrices : ")
```
Remove this line.

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
+
+
+def addspm(sp1, sp2):
+    print("performing addition on given sparse matrices......... ")
```
Remove this line.

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
+        return "addition operation is not possible."
+
+
+# trying to enhance my aproach from above code.
```
Can you please add some more comments to better explain your approach?

atarax665 commented on Python/ds/Sparse_matrix_Operation.py:
```diff
@@ -1,115 +1,165 @@
+# substraction of sparse matrix program
+
+
+def substractspm(sp1, sp2):
```
Add this function in the driver code.

Yash202000 replied:
https://github.com/TesseractCoding/NeoAlgo/commit/55f2531f4b33cc8eb153d3fda6636bede17f648c

I made some changes as you descripted. plz review it once.

Yash202000 committed:
Update Sparse_matrix_Operation.py

added all documentation and corrected  all given directions. added time complexity.

Yash202000 committed:
Update Sparse_matrix_Operation.py

some blank spaces removed.

Yash202000 committed:
Update Sparse_matrix_Operation.py

final changes made.

Yash202000 committed:
Update Sparse_matrix_Operation.py

description added on top

Yash202000 commented:
plz review the changes made in code
commit id :- https://github.com/TesseractCoding/NeoAlgo/commit/55f2531f4b33cc8eb153d3fda6636bede17f648c

atarax665 reviewed (commented):
Remove all the extra blank lines. Format your code according to PEP8 rules. You can use [this](http://pep8online.com/).

HarshCasper commented:
@Yash202000 Please make the README updates as well, and once you are done, squash your commits. We can review it then, because having a lot of commits makes review hard for us.

HarshCasper commented:
Closing the PR because of no updates!
